packages/server/api/src/app/app-connection/app-connection-service/app-connection-service.ts

```diff
@@ -443,6 +443,7 @@ function needRefresh(connection: AppConnection): boolean {
   switch (connection.value.type) {
     case AppConnectionType.PLATFORM_OAUTH2:
     case AppConnectionType.CLOUD_OAUTH2:
+      return oauth2Util.isExpired(connection.value);
```

Comment:
this is mostly the important change

Reply:
you don't need this, the switch has a fallthrough